static/js/components/dashboard/FinancialAidCard_test.js

```diff
@@ -122,7 +122,7 @@ describe("FinancialAidCard", () => {
       program.financial_aid_user_info.has_user_applied = false;
       let wrapper = renderCard({ program });
       wrapper.find('.full-price').simulate('click');
-      assert.ok(setSkipDialogStub.calledWith(true), 'Dialog should get opened');
+      sinon.assert.calledWith(setSkipDialogStub, true);
```

Comment:
Are you talking about the customizable message? [Sinon's assertions](http://sinonjs.org/releases/v1.17.7/assertions/) provide error messages that are also very helpful, although perhaps a customized error message should be _more_ helpful. I'd be happy to revert the changes where I removed a customized error message, but Sinon's default error messages are going to be more helpful and informative than the default `assert.ok` error message.

Is that OK? Revert the changes where I'm removing a customized error message, but keep the changes where there was no customized error message to begin with?